robotd/visiond/camera_base.py

```diff
@@ -0,0 +1,21 @@
+
+class CameraBase:
+    def __init__(self):
+        self.initialised = False
+        self.cam_image_size = None
+
+    def init(self):
+        """ Initialise the camera"""
+        self.initialised = True
+
+    def get_image_size(self):
+        if not self.initialised:
+            raise Exception("Must Initialise camera before getting image size")
+        return self.cam_image_size
+
+    def capture_image(self):
+        """
+        :return: PIL Image captured
+        """
+        pass
```

Comment:
Would be better to raise an explicit `NotImplementedError` in a case like this rather than failing silently.